orb-ui/src/dbus.rs

```diff
@@ -22,7 +22,7 @@ impl Interface {
     /// Forward events to UI engine by sending serialized engine::Event to the event channel.
     async fn orb_signup_state_event(&mut self, event: String) -> zbus::fdo::Result<()> {
         // parse event to engine::Event using json_serde
-        tracing::debug!("received JSON event: {}", event);
+        tracing::trace!("received JSON event: {}", event);
```

Comment:
Do we have tracing enabled? I think given the big changes we do atm it's important to get max info.

Reply:
The solution is probably to change RUST_LOG in the .service file for now. The code should be written with the expectation that anything below INFO is omitted, DEBUG is not high frequency, and TRACE is full verbosity of all everything.

Reply:
parsed events are printed as `debug` logs in the Pearl and Diamond engines, this should be kept as `trace`